device/globals/game.gd

```diff
@@ -55,6 +58,9 @@ func mouse_exit(obj):
 
 func clear_action():
 	current_tool = null
+	# XXX: Should current_action be cleared also for verb menus? Until now it has not been.
```

Comment:
I don't think it should be. With an action menu, that's OK, since the actions are always next to your cursor, but with a verb menu, you'd have to go all the way back to the verb menu to pick the verb again every time it's cleared. Maniac Mansion and Zak McKracken did that, but I always found that more than a little annoying when playing those games. Later LucasArts games did not clear the action.